app/views/internal/shared/_navbar.html.erb

```diff
@@ -1,9 +1,34 @@
-<% menu_items = %w[comments articles users tags welcome broadcasts reports response_templates pages tools chat_channels permissions growth mods config events badges organizations sponsorships podcasts negative_reactions moderator_actions].sort.each_with_object({}) { |v, h| h[v] = v } %>
-<% menu_items[:listings] = "listings" %>
-<% menu_items[:webhooks] = "webhook_endpoints" %>
+<% menu_items = %w[
+     articles
+     badges
+     broadcasts
+     chat_channels
+     comments
+     config
+     events
+     growth
+     listings
+     moderator_actions
+     mods
+     negative_reactions
+     organizations
+     page_redirects
+     pages
+     permissions
+     podcasts
+     reports
+     response_templates
+     sponsorships
+     tags
+     tools
+     users
+     welcome
+   ].each_with_object({}) { |v, h| h[v] = v } %>
+
+<% menu_items["webhooks"] = "webhook_endpoints" %>
```

Comment:
- Added `page_redirects`
- Moved `listings` inside the Array
- Alphabetized the order in the Array
- Broke out each menu item on its own line because 1) Codeclimate was complaining about line length and 2) it's more readble.